backend-lambda/index.js

```diff
@@ -28,7 +28,9 @@ export const handler = async (event, context) => {
   if (event.path === "/products") {
     // get all products
     if (event.httpMethod === "GET") {
-      response = await productService.getAllProducts(pool);
+      const limit = event["multiValueQueryStringParameters"]["limit"][0];
```

Comment:
I think you should be able to access it at `event.queryStringParameters.limit`. Let's check the logs when we talk in a minute.